README.rst

```diff
@@ -53,7 +53,7 @@ And to get the **latest development version**, you can use:
 
 .. code-block::
 
-    $ pip install -U https://github.com/benchopt/benchopt/archive/master.zip
+    $ pip install -i https://test.pypi.org/simple/ benchopt
```

Comment:
```suggestion
    $ pip install -U -i https://test.pypi.org/simple/ benchopt
```
@tomMoral I agree. The `-U` is still needed in case `benchopt` is already installed.